packages/docs/src/locales/en/en.json

```diff
@@ -2981,86 +2980,20 @@
-          "`.align-end` - cross-end margin edge of the items is placed on the cross-end line",
-          "`.align-center` - items are centered in the cross-axis",
-          "`.align-baseline` - items are aligned the way their baselines align"
-        ]
-      },
-      "alignSelf": {
-        "title": "`align-self`:",
-        "variants": [
-          "`.align-self-start` - cross-start margin edge of the items is placed on the cross-start line",
-          "`.align-self-end` - cross-end margin edge of the items is placed on the cross-end line",
-          "`.align-self-center` - items are centered in the cross-axis",
-          "`.align-self-baseline` - items are aligned the way their baselines align"
-        ]
-      },
-      "alignContent": {
-        "title": "`align-content`:",
-        "variants": [
-          "`.align-content-start` - lines packed to the start of the container",
-          "`.align-content-end` - lines packed to the end of the container",
-          "`.align-content-center` - lines packed to the center of the container",
-          "`.align-content-space-between` - lines evenly distributed; the first line is at the start of the container while the last one is at the end",
-          "`.align-content-space-around` - lines evenly distributed with equal space between them"
-        ]
-      },
-      "justify": {
-        "title": "`justify`:",
-        "variants": [
-          "`.justify-start` - items are packed toward the start line",
-          "`.justify-end` - items are packed toward to end line",
-          "`.justify-center` - items are centered along the line",
-          "`.justify-space-around` - items are evenly distributed in the line, items have equal space around on either end",
-          "`.justify-space-between` - items are evenly distributed in the line; first item is on the start line, last item is on the end line",
-          "`.justify-space-evenly` - items are evenly distributed in the line; items have equal space around them"
-        ]
-      },
-      "justifySelf": {
-        "title": "`justify-self`:",
-        "variants": [
-          "`.justify-self-start` - The item is packed flush to each other toward the start edge of the alignment container in the appropriate axis.",
-          "`.justify-self-end` - The item is packed flush to each other toward the end edge of the alignment container in the appropriate axis.",
-          "`.justify-self-center` - For items that are not children of a flex container, this value is treated as `.justify-self-start`.",
-          "`.justify-self-baseline` - The items are aligned according to baselines"
+    "title": "Grid Layout",
+    "summaryText": "We decided to abandon the concept of creating our own CSS helpers (with the exception of [smart helpers](/styles/smart-helpers)[[target=_blank]]). Therefore, now we recommend our users to use [Tailwind CSS](https://tailwindcss.com/)[[target=_blank]] as the most convenient and modern solution to the issue.",
```

Comment:
That's historic data and not very relevant to new users.

Let's keep deprecated page with old styles (which we'll delete for 1.7.0) and page with tailwind integration separate.

Tailwind page - should only talk about tailwind and which role it fits in current state of things.